tests/components/sensor/test_geo_rss_events.py

```diff
@@ -0,0 +1,137 @@
+"""The test for the geo rss events sensor platform."""
+import unittest
+
+from homeassistant.setup import setup_component
+from tests.common import load_fixture, get_test_home_assistant
+import homeassistant.components.sensor.geo_rss_events as geo_rss_events
+
+URL = 'http://geo.rss.local/geo_rss_events.xml'
+VALID_CONFIG_WITH_CATEGORIES = {
+    'platform': 'geo_rss_events',
+    geo_rss_events.CONF_URL: URL,
+    geo_rss_events.CONF_CATEGORIES: [
+        'Category 1',
+        'Category 2'
+    ]
+}
+VALID_CONFIG_WITHOUT_CATEGORIES = {
+    'platform': 'geo_rss_events',
+    geo_rss_events.CONF_URL: URL
+}
+
+
+class TestGeoRssServiceUpdater(unittest.TestCase):
+    """Test the GeoRss service updater."""
+
+    def setUp(self):
+        """Initialize values for this testcase class."""
+        self.hass = get_test_home_assistant()
+        self.config = VALID_CONFIG_WITHOUT_CATEGORIES
+
+    def tearDown(self):
+        """Stop everything that was started."""
+        self.hass.stop()
+
+    def test_setup_with_categories(self):
+        """Test the general setup of this sensor."""
+        self.config = VALID_CONFIG_WITH_CATEGORIES
+        self.assertTrue(
+            setup_component(self.hass, 'sensor', {'sensor': self.config}))
+        self.assertIsNotNone(
+            self.hass.states.get('sensor.event_service_category_1'))
+        self.assertIsNotNone(
+            self.hass.states.get('sensor.event_service_category_2'))
+
+    def test_setup_without_categories(self):
+        """Test the general setup of this sensor."""
+        self.assertTrue(
+            setup_component(self.hass, 'sensor', {'sensor': self.config}))
+        self.assertIsNotNone(self.hass.states.get('sensor.event_service_any'))
+
+    def test_filter_entries(self):
```

Comment:
Do you still need this test?

Reply:
Probably not; merging with method `test_update_sensor_with_category` to retain test coverage.